packages/react-core/src/components/Wizard/WizardNavItem.tsx

```diff
@@ -26,16 +28,22 @@ export const WizardNavItem: React.FunctionComponent<WizardNavItemProps> = ({
   isDisabled = false,
   step,
   onNavItemClick = () => undefined,
-  navItemComponent = 'a'
+  navItemComponent = Button
```

Comment:
In core they use anchor and this could be a breaking change
https://pf4.patternfly.org/documentation/core/components/wizard

Reply:
Yep, I wanted to use the react workspace to facilitate the conversation on both ends since we can better see the components in action with these interactive examples. If we're good with it we'd definitely need to make corresponding changes in core.

Reply:
Seems like it needs to be configurable between link and button, and maybe favor button as the default since we're typically building SPAs these days. In case the user isn't using a single page app they'll want anchors, and in the case of SPA button seems to carry the most appropriate semantics and behavior as selecting them doesn't currently change the page URL address. We could look into hooking in routing with the nav items, but that's starting to look like a good place to break out into a separate effort.

So yea definitely is missing the anchor support now (I can add this back in with a simple prop to toggle between the two). Does that sound ok? @jschuler

Reply:
If we're supporting both `<a>` and `<button>` here, I don't think that's breaking. Shouldn't be breaking unless we remove support for `a.wizardNavLink`. Though I'm not sure we would use the inline button variation here - we could also just add `border: 0;` (I think that's it?) to the `wizardNavLink` styles to support use of the `<button>` html element.

Reply:
@seanforyou23 opened https://github.com/patternfly/patternfly/pull/3216 if you want to have a look.

Reply:
Made some updates, will remove the temporary inline style soon as we get your core PR merged. Let me know if you see any issues with the styling!

Reply:
Oh I forgot to mention - I noticed this weird behavior in chrome where the focus outline applies to the item's pseudo element (the number), too. We can likely fix that with `:focus-within` on the parent list item. WDYT, is that worth updating?

<img width="177" alt="Screen Shot 2020-06-26 at 9 25 19 AM" src="https://user-images.githubusercontent.com/35148959/85867901-32e07a80-b78f-11ea-9ca9-fcf2056c40dc.png">

Reply:
Hmm good question. Currently the number is announced when you focus the item which seems like it may have been done by design. Would changing to focus-within stop the step numbers from being automatically announced? I think the main thing we'd want to try and preserve is the current behavior in Safari in terms of how the items are announced to SRU

Reply:
I'm not aware of any impacts to screen readers when using `:focus-within`, I believe it just changes the presentation of the focus indicator. I made a couple of demos here if you want to see if there is any sort of effect. The links with "focus within" in the text change the focus indicator so that the number doesn't get its own focus ring.

https://codepen.io/mcoker/pen/pogrRwB - using `:focus-within`
https://codepen.io/mcoker/pen/yLeogaw - just changes the link/button's `outline-style`

Reply:
Yea both of those methods seem to announce the step number just fine. `:focus-within` sounds good to me, good catch!